packages/sirius-web/frontend/sirius-web-application/src/theme/siriusWebTheme.ts

```diff
@@ -78,6 +78,11 @@ export const siriusWebTheme = createTheme(
           backgroundColor: baseTheme.palette.primary.main,
         },
       },
+      MuiTooltip: {
+        tooltip: {
+          backgroundColor: baseTheme.palette.common.black,
+        },
+      },
```

Comment:
This will apply globally, including e.g. for widgets help text. Was this change discussed?

Reply:
Yes.